.github/workflows/release.yml

```diff
@@ -39,15 +39,19 @@ jobs:
           node-version: '14.x'
           registry-url: 'https://registry.npmjs.org'
 
+      # We need llvm-lipo utility from LLVM to build a universal mac binary.
+      - run: wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh && sudo ./llvm.sh 16
```

Comment:
Installing through LLVM rather than ubuntu package, to make sure the version is alinged with the one used by zig (which is usually fresher than what's publsihed with ubuntu)